screens/AddHotspotScreen.tsx

```diff
@@ -184,23 +187,28 @@ const getStyles = (theme: ReactNativePaper.Theme) =>
     },
   });
 
-export const AddHotspotScreen = () => {
+export const AddHotspotScreen = ({
+  route,
+}: RootStackScreenProps<'AddHotspot'>) => {
+  const navigation = useNavigation();
   const theme = useTheme();
   const styles = getStyles(theme);
 
   const [location, setLocation] = useState<Location>();
+  const [name, setName] = useState<string>('');
   const [status, setStatus] = useState<HotspotStatus>();
-  const [showStatusDropDown, setShowStatusDropDown] = useState(false);
+  const [remarks, setRemarks] = useState<string>('');
+  const [contactPerson, setContactPerson] = useState<string>('');
+  const [contactPersonPhone, setContactPersonPhone] = useState<string>('');
+  const [volunteer, setVolunteer] = useState<string>('');
 
-  const statusList: HotspotStatus[] = [
-    HotspotStatus.toDo,
-    HotspotStatus.inProgress,
-    HotspotStatus.done,
-  ];
+  useEffect(() => {
+    setLocation(route.params.location);
+  }, [route.params.location]);
 
   return (
     <View style={styles.container}>
-      <Appbar />
+      <SecondaryAppbar onBackPressed={() => navigation.goBack()} />
```

Comment:
A customised version of the `AppBar` should be used instead of another component.
The behavior `navigation.goBack()` for the back button should be the default and it should not be specified as a prop.

Reply:
done.